src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -18,11 +18,11 @@
     @Schema(enumeration = { "webhook", "email_subscription", "default" })
     public enum EndpointType {
         @JsonProperty("webhook")
-        WEBHOOK,
+        WEBHOOK, // 0
```

Comment:
Add a comment on top of the enum, that new values must be added at the end.

Reply:
Done